src/DotVVM.Samples.Common/Utilities/JavaScriptUtils.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using DotVVM.Framework.Utils;
+
+namespace DotVVM.Samples.Common.Utilities
+{
+    public static class JavaScriptUtils
+    {
+        public static string LimitLength(string text, int length) => StringUtils.LimitLength(text, length);
```

Comment:
Why not just use the StringUtils.LimitLength in the value binding?

Reply:
And what about to add support for this function to framework?

Reply:
> Why not just use the StringUtils.LimitLength in the value binding?

I wanted to have a proxy class for these javascript utilities functions

> And what about to add support for this function to framework?

I'm not sure about adding support in the framework, instead I would like to make the **javascript method registration** a lot easier.